.travis.yml

```diff
@@ -45,8 +44,8 @@ script:
   - cat cov1.out >> coverage.txt
   - cat cov2.out >> coverage.txt
   - cd /jfs
-  - git clone https://github.com/sanwan/gofslock.git
-  - cd gofslock
+  - git clone https://github.com/danjacques/gofslock.git
+  - cd gofslock && git reset --hard cc7f001fe0e7df1710adc8f0cd9e9d6d21fdb3a9
```

Comment:
`git checkout cc7f001fe0e7df1710adc8f0cd9e9d6d21fdb3a9` might be more proper

Reply:
ok